tests/reconfiguration.py

```diff
@@ -149,6 +149,55 @@ def test_add_node(network, args, from_snapshot=True):
     return network
 
 
+@reqs.description("Test CCF_IGNORE_FIRST_SIGTERM")
+def test_ccf_ignore_first_sigterm(network, args):
+    # Note: host is supplied explicitly to avoid having differently
+    # assigned IPs for the interfaces, something which the test infra doesn't
+    # support widely yet.
+    operator_rpc_interface = "operator_rpc_interface"
+    host = infra.net.expand_localhost()
+    new_node = network.create_node(
+        infra.interfaces.HostSpec(
+            rpc_interfaces={
+                infra.interfaces.PRIMARY_RPC_INTERFACE: infra.interfaces.RPCInterface(
+                    host=host
+                ),
+                operator_rpc_interface: infra.interfaces.RPCInterface(
+                    host=host,
+                    endorsement=infra.interfaces.Endorsement(
+                        authority=infra.interfaces.EndorsementAuthority.Node
+                    ),
+                ),
+            }
+        )
```

Comment:
Is this needed?

Reply:
No

Reply:
It is apparently, and there's not much reason to have it different than the rest of the test, so keeping it that way.

Reply:
It's odd that this is needed at all. What is the error when this is commented out?